readme.md

```diff
@@ -90,66 +90,77 @@ https://webapps.stackexchange.com/questions/74859/what-format-does-word-online-u
-          --logfilename
-      -Q  Quiet Mode: Nothing will be output to console.  To see any errors you must 
-          set -G or -GL. Equivalent to setting -L 0
-          --quiet
-      -R  Remove Files after successful conversion: Default false; To use specifiy value eg -R true
-          --deletefiles
-      -W  Webhook: Url to call on events. See -HW for more details.
-          --webhook
-      -HW Webhook Help.
-      -X  Halt on COM Error: Default True;  If you have trouble with some files 
-          not converting, set this to false to ignore errors and continue with batch job.
-          --halterror
-      -V  Show Versions.  DocTo and Word/Excel
+	  -H   This message
+		   --HELP -?
+	  -XL  Use Excel for Conversion
+		   --excel
+	  -WD  Use Word for Converstion (Default)
+		   --word
+	  -F   Input File or Directory
+		   --inputfile
+	  -FX  Input Extension to search for if directory.
+		   Default "*.doc*" (will find ".docx" also)
+		   --inputextension
+	  -O   Output File or Directory to place converted Docs
+		   --outputfile
+	  -OX  Output Extension if -F is Directory. Please include '.' eg. '.pdf' .
+		   If not provided, pulled from standard list.
+		   --outputextension
+	  -T   Format(Type) to convert file to, either integer or wdSaveFormat constant.
+		   Available from
+		   https://docs.microsoft.com/en-us/dotnet/api/microsoft.office.interop.word.wdsaveformat
+		   or https://docs.microsoft.com/en-us/dotnet/api/microsoft.office.interop.excel.xlfileformat
+		   See current List Below.
+		   --format
+	  -TF  Force Format. -T value if an integer, is checked against current list
+		   compiled in. It is not passed if unavailable.  -TF will pass through value
+		   without checking. Word will return an "EOleException  Value out of range"
+		   error if invalid.
+		   Use instead of -T.
+		   --forceformat
+	  -L   Log Level Integer: 1 ERRORS Only, 2 STANDARD, 5 CHATTY, 9 DEBUG,
+		   10 VERBOSE.  Default: 2=STANDARD
+		   --loglevel
+	  -C   Compatibility Mode Integer. Set to an INTEGER value from
+		   https://msdn.microsoft.com/en-us/library/office/ff192388.aspx.
+		   Set the compatibility mode when you want to convert documents to a later
+		   version of word. See List Below
+		   --compatibility
+	  -E   Encoding Integer: Sets codepage Encoding.  See
+		   https://msdn.microsoft.com/en-us/library/office/ff860880.aspx
+		   for more details and values.
+		   --encoding
+	  -M   Ignore all files in __MACOSX\ subdirectory if it exists.  Default True.
+		   --ignoremacos
+	  -N   Make list of files that take over n seconds to complete.
+		   Use number of seconds over that conversion takes and add to list.
+		   Outputs to filename 'docto.ignore.txt'
+		   --listlongrunning
+	  -NX  Ignore any file listed in docto.ignore.txt, created by -N
+		   --ignorelongrunninglist
+	  -OAE Open after export: If set, the created PDF file (or other format) will be opened in the associated application. Default False. 
```

Comment:
I made a recent decision that most new parameters are going to be long form only unless they really deserve short form.  This is similar to the recently added --bookmarksource

I think rather than `-OAE` we should use `--pdf-openafterexport` .  i think the pdf part is important as it is relevant only to exporting as pdf.